atomic_reactor/utils/imageutil.py

```diff
@@ -138,7 +138,8 @@ def extract_file_from_image(self, image: Union[str, ImageName],
         if any(Path(dst_path).iterdir()):
             raise NonEmptyDestinationError(f'the destination directory {dst_path} must be empty')
 
-        cmd = ['oc', 'image', 'extract', f'{image}', '--path', f'{src_path}:{dst_path}']
+        cmd = ['oc', 'image', 'extract', f'{image}', '--confirm', '--path', f'{src_path}'
+                                                                            f':{dst_path}']
```

Comment:
Is the last argument supposed to be split like that?

Reply:
updated